src/sentry/newsletter/base.py

```diff
@@ -0,0 +1,20 @@
+from __future__ import absolute_import
+
+
+class Newsletter(object):
+    PRODUCT_CAMPAIGN = 1
```

Comment:
this would probably be better stored as a string so its more flexible in the future